app/models/medical_history.rb

```diff
@@ -35,4 +35,8 @@ def indicates_risk?
       diabetes_boolean ||
       chronic_kidney_disease_boolean
   end
+
+  def indicates_hypertension_risk?
```

Comment:
Since we have this, I believe `indicates_risk?` can go away (along with tests).

Reply:
Ah, coolbeans. I thought there might be merit in keeping it around, but you're right. If it's currently unused, then we should axe it (and tests)